Core/include/Acts/Navigation/DetectorNavigator.hpp

```diff
@@ -146,14 +146,24 @@ class DetectorNavigator {
     auto& nState = state.navigation;
 
     if (nState.currentDetector == nullptr) {
-      ACTS_VERBOSE("assigning detector from the config.");
+      ACTS_VERBOSE("Assigning detector from the config.");
       nState.currentDetector = m_cfg.detector;
     }
-
     if (nState.currentDetector == nullptr) {
-      ACTS_ERROR("panic: no detector");
+      ACTS_VERBOSE("Initialization: no detector assigned");
```

Comment:
is this not an error?

Reply:
It is, but maybe we should throw an exception somewhere at the construction? I just discovered that there is a logging level failure threshold, so this one is temporary, just to check that the other tests are fine. Working on the error handling in the meantime